static/js/topic_generator.js

```diff
@@ -80,14 +80,19 @@ exports.wrap_exclude = function (lst, val) {
     if (val === undefined) {
         return exports.list_generator(lst);
     }
-
-    var i = _.indexOf(lst, val);
+    var i;
+    for (i in lst) {
+        if (lst[i].stream === val.stream && lst[i].topic === val.topic) {
+            break;
+        }
+        i = -1;
+    }
+    i = parseInt(i, 10);
```

Comment:
I went through some of the stackoverflow answers, this is almost what most of them have mentioned. The actual problem is comparing the dict objects, which can be done only by ` lst[i].stream === val.stream && lst[i].topic === val.topic`